packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -90,10 +88,40 @@ export default class QuanticResultLink extends LightningElement {
     );
   };
 
+  handleOpenTabSalesforceLink(event) {
+    event.stopPropagation();
+    const targetPageRef = {
+      type: 'standard__recordPage',
+      attributes: {
+        recordId: this.getRecordIdAttribute(),
+        actionName: 'view',
+      },
+    };
+    this[NavigationMixin.Navigate](targetPageRef);
+  }
+
+  getRecordIdAttribute() {
+    let idToUse = this.result.raw.sfid;
+
+    // Knowledge article uses the knowledge article version id to navigate.
+    if (this.result?.raw?.sfkbid && this.result?.raw?.sfkavid) {
+      idToUse = this.result.raw.sfkavid;
+    }
+    return idToUse;
+  }
```

Comment:
a bit more simple imo
```suggestion
  get recordId() {
    // Knowledge article uses the knowledge article version id to navigate.
    if (this.result?.raw?.sfkbid && this.result?.raw?.sfkavid) {
      return this.result.raw.sfkavid;
    }
    return this.result.raw.sfid;
  }
```